www/src/pages/api/login.ts

```diff
@@ -0,0 +1,57 @@
+import type { APIRoute } from 'astro';
+import { z } from 'zod';
+import { User } from '@backend/user';
+
+const loginErrors = {
+    email: {
+        empty: 'Email cannot be empty',
+        invalid: 'Enter valid email',
+    },
+    password: 'Password cannot be empty',
+    form: 'Invalid credentials',
+} as const;
```

Comment:
We really don't need to provide much info in the response in this case. We don't care if email has wrong syntax or password is empty. We should just return that login failed and that's it.